### PR TITLE
add command and link for elasticsearch

### PR DESCRIPTION
**Pull request by CharlesSheelam**
This pull request updates the environment setup instructions on the readme.md file,  and adds some additional info for step 3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Pull requests, issues to request new features, enhancements, or bug reports are
 -------------
 1. [Install Docker](https://docs.docker.com/engine/install/) ([Ubuntu-specific install instructions](https://docs.docker.com/desktop/install/ubuntu/))
 2. The new `docker-compose-plugin` (e.g., `$ apt install -y docker-compose-plugin`) provides a `docker compose ...` command that replaces old `docker-compose ...` invocations
-3. Add `vm.max_map_count=262144` to `/etc/sysctl.d/99-docker.conf` for elasticsearch
+3. Add `vm.max_map_count=262144` to `/etc/sysctl.d/99-docker.conf` for elasticsearch. Access the file with the command : (`$ sudo nano /etc/sysctl.d/99-docker.conf`). More info on (https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html)
```
Comment (alee) [inline README.md:72]: How about something like:

3. Create or update a file, `/etc/sysctl.d/99-docker.conf` with the line `vm.max_map_count=262144` so [elasticsearch can run properly](https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html). You can create / access the file with an invocation like `$ sudo nano /etc/sysctl.d/99-docker.conf` and following the in-terminal instructions to save and exit.

**Review by alee — CHANGES_REQUESTED** [review]
Hi Charles, thanks for the PR! Got a few minor text changes to add to this PR, but you should also ensure that [maintainers can edit your PR / branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so we can help you get changes merged.

**Comment by CharlesSheelam** [comment]
Thanks Allen! Going to make those changes now, also will remember to switch on allow edits by maintainers for the future.

**Review by alee — COMMENTED** [review]
Looks like the link to the actual additional context got lost in the subsequent commits but I'll go ahead and squash & merge this and manually the docs changes to the README so we can get to actual coding

**Pull request merged**